File: Python/day_3_register_check.py
def register_check(object):
    print(object)
    count=0
    for i in object:
        if (object.get(i)=="yes") or (object.get(i)=="Yes"):
            count+=1
    return count
para={'Michael':'Yes','John':'yes','Peter':'no','Mary':"yes"}
output=register_check(para)
print(output)

File: JavaScript/day_3_register_check.js
function register_check(objectt){
    let count=0;
    for(let xn in objectt){
        if((objectt[xn]=="yes")||(objectt[xn]=="Yes")){
            count+=1;
        }
    }
    return count;
}
let para={'Michael':'Yes','John':'yes','Peter':'no','Mary':"yes"};
let output=register_check(para);
console.log(output);